Sources/Kitura/Headers.swift

```diff
@@ -66,24 +76,35 @@ extension Headers: Collection {
         }
     }
     
+    /// Get a (key value) tuple from the `Headers` collection at the specified position.
+    ///
+    /// - Parameter position: The position in the `Headers` collection of the (key, value)
+    ///                      tuple to return.
+    ///
+    /// - Returns: A (key, value) tuple.
     public subscript(position: HeadersIndex) -> (String, String?) {
         get {
             let (key, value) = headers[position]
             return (key, value.first)
         }
     }
     
+    /// Get the next Index in the `Headers` collection after the one specified.
+    ///
+    /// - Parameter after: The Index before the Index to be returned.
```

Comment:
Instead changed line 94 to read:

Parameter after: The Index who's successor is to be returned.